clients/web/test/testUtils.js

```diff
@@ -1278,10 +1278,11 @@ girderTest.startApp = function () {
                 parentView: null,
                 start: false
             });
+            girder.app = app;
             return app.start();
         })
         .then(function () {
-            girder.events.trigger('g:appload.after');
+            girder.events.trigger('g:appload.after', app);
```

Comment:
It'd be simpler to just reference `girder.app` here, then remove the `var app` from the parent scope entirely.